.travis.yml

```diff
@@ -44,7 +44,9 @@ matrix:
     name: "E2E tests"
     if: branch !~ /(^branch-.*-built)/ AND env(RUN_E2E) = true
     before_script:
-    - nvm install
+    # Enable user namespace cloning
+    - "sysctl kernel.unprivileged_userns_clone=1"
+    - nvm install 10
```

Comment:
Doesn't nvm file already define 10 also for Travis?

https://github.com/Automattic/jetpack/blob/master/.nvmrc

Reply:
addressed in 476f303